content/docs/guides/crossguard/configuration.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Configurable Policy Packs
+meta_desc: Configuration allows you to write flexible policies that can be reused across you organization.
+linktitle: Configuration
+weight: 2
+
+menu:
+  userguides:
+    parent: crossguard
+---
+<!-- markdownlint-disable ul code -->
+{{% crossguard-preview %}}
+
+## Overview
+
+Configuration allows you to author flexible Policy Packs that can be reused across your entire organization. As an organization administrator, you can use a single Policy Pack yet vary configuration (e.g. enforcement level, the allowed instance types, cost allowances, etc.) from one Policy Group to the next. For example, you may have a Policy Group for your non-production stacks that allow smaller instance types, while your Production Policy Group allows for use of large instance types.
+
+Configuration schema is defined per policy, and then the actual configuration can be set via a form in the Pulumi Console or using a JSON file.
+
+Configuration is currently supported for the TypeScript and JavaScript Policy SDKs. Python support for configuration is [coming soon](https://github.com/pulumi/pulumi-policy/issues/210).
+
+## Writing Configurable Policy Packs
+
+### Enforcement Level
+
+The Policy SDK will automatically make the enforcement levels of all policies configurable. You can configure the enforcement level for all policies in a Policy Pack or configure it for each individual policy level. The top level enforcement level under `all` will override the existing enforcement levels of all policies in the Policy Pack. A configured enforcement level for a specific policy will override the policy's existing enforcement level as well as the top level configured enforcement level.
+
+In the example configuration below, all policies in the Policy Pack would be `disabled` with the exception of the `a-policy` policy, which would be `mandatory`.
+
+```json
+{
+    "all": {
+        "enforcementLevel": "disabled",
+    },
+    "a-policy": {
+         "enforcementLevel": "mandatory",
+    }
+}
+```
```

Comment:
JSON can be more strict about trailing commas. I don't know off the top of my head if Go's JSON unmarshalling is lenient in this regard, but I'd personally leave them off.

````suggestion
```json
{
    "all": {
        "enforcementLevel": "disabled"
    },
    "a-policy": {
         "enforcementLevel": "mandatory"
    }
}
```
````